staff/almu-itur/cv-styled/styles/styles.css

```diff
@@ -0,0 +1,282 @@
+* {
+    font-family: 'Raleway', sans-serif;
+}
+
+@media screen and (min-device-width: 481px) and (max-device-width: 768px) and (orientation:portrait) {
```

Comment:
I think it would be a good idea to eliminate  "and (min-device-width: 481px)" and add above the MediaQuerys:
body{
min-width:480px;
}